src/features/CheckoutPage/components/PaymentInformation.jsx

```diff
@@ -0,0 +1,81 @@
+
+import React, {useState, Fragment} from 'react';
+import {Listbox} from '@headlessui/react';
+import {AiOutlineCheck} from 'react-icons/ai';
+
+const PaymentInformation = ({submitHandler}) => {
+  const [name, setName] = useState('');
+  const [number, setNumber] = useState('');
+  const [month, setMonth] = useState('January');
+  const [year, setYear] = useState('2025');
+  const [cvv, setCVV] = useState('');
+  const months = ['January', 'Febuary', 'March', 'April', 'May',
+    'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+  const onFormInputChange = async (e) => {
+    if (e.target.name == 'name') setName(e.target.value);
+    else if (e.target.name == 'number') setNumber(e.target.value);
+    else if (e.target.name == 'month') setMonth(e.target.value);
+    else if (e.target.name == 'year') setYear(e.target.value);
+    else if (e.target.name == 'cvv') setCVV(e.target.value);
```

Comment:
add mapping of key value pair name <=> action and use that instead of multiple if else